spec/openapi.yaml

```diff
@@ -6468,6 +6468,84 @@ components:
             The subject field is used to denote subject of the MMS message and
             has a maximum size of 64 characters long
           example: New Products
+    DetailMessage:
```

Comment:
Suggestion, rename this `DetailReportMessageRecord` to avoid any misuse, editing, or confusion.